src/style.css

```diff
@@ -0,0 +1,16 @@
+* {
```

Comment:
rename this file to normalize.css, add file style.css import all CSS file there, and connect to HTML page just style.css file